Soccer Match.rtf

```diff
@@ -0,0 +1,18 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2580
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 The app we\'92re building is soccer game. we need 2 teams of 11 players. they will play in a match. the match will keep the score. at the end we will display who wins.\
+\
+class Player \{\
+var goals: Int\
+let name: String\
+let country: String\
+let age: Int\
+weak var team: Team\
```

Comment:
btw, `weak` relationship between this object and the `Team` object means one doesn't depend on the other, so if this `Player` is ever "freed" for whatever reason (if he quits or is sold to another team, retires, etc) the team can let him go. Parent/child relationships should always have this type of relationship if they reference each other though. 

also you'll be creating the team object